discord/ext/commands/converter.py

```
@@ -822,6 +824,27 @@ async def convert(self, ctx: Context, argument: str) -> discord.PartialEmoji:
 
         raise PartialEmojiConversionFailure(argument)
 
+class FormattedDatetimeConverter(Converter[Tuple[datetime.datetime, Optional[str]]]):
+
+    """Converts a discord style datetime to a :class:`datetime.datetime`.
+    Also returns the flag used to format the timestamp if present.
+
+    This is done by extracting the epoch from the string.
+
+    .. versionadd:: 2.0
+        Raise :exc:`.FormattedDatetimeConversionFailure` instead of generic :exc:`.BadArgument`
+     """
+
+    async def convert(self, ctx: Context, argument: str) -> Tuple[datetime.datetime, Optional[str]]:
+        match = re.match(r'<t:(-?\d+)(:[tTdDfFR])?>$', argument)
+
+        if match:
+            try:
+                return datetime.datetime.fromtimestamp(int(match.group(1))), match.group(2)
+            except:
```

You shouldn't be using a bare except